shell.sh

```
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+API_KEY="hardcoded_api_key"
+
+unused_var="This variable is never used"
```

Shellcheck

[warning] unused_var appears unused. Verify use (or export if used externally).

(SC2034)